clj-refactor.el

```diff
@@ -1983,37 +1988,47 @@ the alias in the project."
     (backward-sexp 1)
     (looking-at-p "[-+0-9]")))
 
+(defun completable-for-cljr-slash? (sym)
+  (when sym
+    (or
+     ;; vanilla symbols, maybe prefixed by ^, :: or ^::
+     (not (null (string-match-p "^\\^?\\(::\\)?\\([a-zA-Z]+[a-zA-Z0-9\\-]*\\)+\\(\\.?[a-zA-Z]+[a-zA-Z0-9\\-]*\\)*$" sym)))
```

Comment:
Seems to me you can rewrite this code with the not null check and it'd read better.

Reply:
I cargo-culted the pattern from https://github.com/magnars/s.el/blob/08661efb075d1c6b4fa812184c1e5e90c08795a9/s.el#L281-L283 (`s--truthy?` is used a few times there, so transitively the not-null check is used a bunch of times there)

No strong opinion from my side, but maybe if in doubt we can do as s.el does

Reply:
(bump)

Reply:
I can live with this, it just seems quite verbose.